src/app/containers/manuscript/references-list/index.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { useCallback } from 'react';
+import { get } from 'lodash';
+import { useSelector } from 'react-redux';
+import { IconButton } from '@material-ui/core';
+import EditIcon from '@material-ui/icons/Edit';
+
+import { SectionContainer } from 'app/components/section-container';
+import { ActionButton } from 'app/components/action-button';
+import { getReferences } from 'app/selectors/manuscript.selectors';
+import { Reference } from 'app/models/reference';
+import { useReferencesListItemStyles, useReferencesListStyles } from 'app/containers/manuscript/references-list/styles';
+import { stringifyEditorState } from 'app/utils/view.utils';
+
+const getAnnotatedText = (reference: Reference, path: string) => {
+  const editorState = get(reference, path);
+  if (editorState) {
+    const html = stringifyEditorState(editorState);
+    return <span dangerouslySetInnerHTML={{ __html: html }} />;
```

Comment:
I'll take it there isn't really any other choice here? Not being a React expect, but can you not just do...
`<span>{html}</span>`?

Reply:
when you do it that way the value is automatically sanitized. The `dangerouslySetInnerHTML` is the way to skip sanitization